osu.Game.Tests/Visual/Gameplay/TestSceneMultiplayerGameplayLeaderboard.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Utils;
+using osu.Game.Database;
+using osu.Game.Online.Spectator;
+using osu.Game.Replays.Legacy;
+using osu.Game.Rulesets.Osu.Scoring;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Scoring;
+using osu.Game.Screens.Play.HUD;
+using osu.Game.Tests.Visual.Online;
+
+namespace osu.Game.Tests.Visual.Gameplay
+{
+    public class TestSceneMultiplayerGameplayLeaderboard : OsuTestScene
+    {
+        [Cached(typeof(SpectatorStreamingClient))]
+        private TestMultiplayerStreaming streamingClient = new TestMultiplayerStreaming(16);
+
+        [Cached(typeof(UserLookupCache))]
+        private UserLookupCache lookupCache = new TestSceneCurrentlyPlayingDisplay.TestUserLookupCache();
+
+        [SetUp]
+        public void SetUp() => Schedule(() =>
+        {
+            OsuScoreProcessor scoreProcessor;
+
+            streamingClient.Start(Beatmap.Value.BeatmapInfo.OnlineBeatmapID ?? 0);
+
+            Children = new Drawable[]
+            {
+                streamingClient,
+                lookupCache,
```

Comment:
If `SetUp()` runs more than once, these are getting disposed the second time this `Children` assignment runs (as it's a shorthand for `Clear()` followed by an `AddRange()`), causing the other test failure.